### PR TITLE
M0 mr intra pred

### PR DESCRIPTION
**Pull request by OmranAbdelkafi**
**Description**
Adopt MR_Mode's Intra_pred in encoder mode 0 : 

**Improvements**
~ 0.2% BD-rate gain over the full list

**Speed Impact**
Using AWS instance (72 cores) : ~4.3% speed drop

### CODE REVIEW

File: Source/Lib/Common/Codec/EbPictureDecisionProcess.c [hunk 1]
```diff
@@ -1443,7 +1443,9 @@ EbErrorType signal_derivation_multi_processes_oq(
 #if NEW_PRESETS
 #if SCREEN_CONTENT_SETTINGS
     if (sc_content_detected)
-        if (picture_control_set_ptr->enc_mode <= ENC_M2)
+        if (picture_control_set_ptr->enc_mode == ENC_M0) 
+			picture_control_set_ptr->intra_pred_mode = 0;
```
Comment (hassount) [inline Source/Lib/Common/Codec/EbPictureDecisionProcess.c]: no tabs

File: Source/Lib/Common/Codec/EbPictureDecisionProcess.c [hunk 2]
```diff
@@ -1458,7 +1460,9 @@ EbErrorType signal_derivation_multi_processes_oq(
     else
 
 #endif
-        if (picture_control_set_ptr->enc_mode  <= ENC_M1)
+        if (picture_control_set_ptr->enc_mode == ENC_M0)
+			picture_control_set_ptr->intra_pred_mode = 0;
```
Comment (hassount) [inline Source/Lib/Common/Codec/EbPictureDecisionProcess.c]: no tabs

File: Source/Lib/Common/Codec/EbPictureDecisionProcess.c (hunk 1 above)
Comment (1480c1) [reply Source/Lib/Common/Codec/EbPictureDecisionProcess.c]: @OmranAbdelkafi if you can use sed, you can use `sed -i 's/\t/    /g' Source/Lib/Common/Codec/EbPictureDecisionProcess.c`

File: Source/Lib/Common/Codec/EbPictureDecisionProcess.c [hunk 3]
```diff
@@ -1458,7 +1460,9 @@ EbErrorType signal_derivation_multi_processes_oq(
     else
 
 #endif
-        if (picture_control_set_ptr->enc_mode  <= ENC_M1)
+        if (picture_control_set_ptr->enc_mode == ENC_M0)
+	        picture_control_set_ptr->intra_pred_mode = 0;
```
Comment (1480c1) [inline Source/Lib/Common/Codec/EbPictureDecisionProcess.c]:
```suggestion
            picture_control_set_ptr->intra_pred_mode = 0;
```
this is still a tab

**Pull request merged**